crates/sqlite/build.rs

```diff
@@ -67,7 +67,7 @@ fn main() {
     .define("SQLITE_ENABLE_FTS3_PARENTHESIS", "1")
     .define("SQLITE_ENABLE_FTS5", "1")
     .define("SQLITE_ENABLE_LOAD_EXTENSION", "1")
-    .define("SQLITE_ENABLE_MATH_FUNCTIONS", "1")
+    .define("SQLITE_ENABLE_MATH_FUNCTIONS", "0")
```

Comment:
we have to link to `libm` anyway, it turns out, so this can be enabled with very low cost